src/scripts/components/Wallet.js

```diff
@@ -1,13 +1,28 @@
 import React, { Component } from 'react'
 import Card from 'components/structures/Card'
 import Text from 'components/foundations/Text'
+import Button from './foundations/Button'
 import paratii from 'utils/ParatiiLib'
 
 type Props = {
-  user: Object
+  user: Object,
+  closeModal: () => void,
+  openModal: () => void
 }
 
 class Wallet extends Component<Props> {
+  secureWallet: (e: Object) => void
+
+  constructor (props: Props) {
+    super(props)
+    this.secureWallet = this.secureWallet.bind(this)
+  }
+
+  secureWallet () {
+    console.log('secure')
+    this.props.openModal('ModalSecure')
```

Comment:
It's not necessary to do, but if you find yourself with extra time (😜) it would be awesome to not use raw strings and instead move all modal names to a new `constants/ModalConstants.js` file and define all modal names/import them from there.